src/routes/alerts/alerts.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { JsonSchemaService } from '@/validation/providers/json-schema.service';
+import { AlertsController } from '@/routes/alerts/alerts.controller';
+import { AlertsService } from '@/routes/alerts/alerts.service';
+
+@Module({
+  providers: [JsonSchemaService, AlertsService],
```

Comment:
`JsonSchemaService` can be removed as it is provided by a global module so it's already available to this module.

Reply:
I've removed it in ffbf4a04e7995592f9eeb14a12cfb84342b54672.